examples/dynamic-bridge-app/linux/DynamicDevice.cpp

```diff
@@ -17,12 +17,27 @@
  *    limitations under the License.
  */
 
+#include <app-common/zap-generated/af-structs.h>
+#include <app-common/zap-generated/attribute-id.h>
+#include <app-common/zap-generated/cluster-id.h>
```

Comment:
This is just right. We should not be including code-generated files from common library code.